packages/csharp/ArmoniK.Api.Worker/Utils/WorkerServer.cs

```diff
@@ -48,6 +48,18 @@ namespace ArmoniK.Api.Worker.Utils;
 [PublicAPI]
 public static class WorkerServer
 {
+  /// <summary>
+  ///   Create a web application for the given ArmoniK Worker gRPC Service
+  /// </summary>
+  /// <typeparam name="T">gRPC Service to add to the web application</typeparam>
+  /// <returns>
+  ///   The web application initialized
+  /// </returns>
+  public static WebApplication Create<T>()
+    where T : gRPC.V1.Worker.Worker.WorkerBase
+    => Create<T>(null);
```

Comment:
This should be handled by the following function:

```c#
  public static WebApplication Create<T>(IConfiguration?             configuration       = null,
                                         Action<IServiceCollection>? serviceConfigurator = null)
```

Reply:
So you should most likely remove the optional first parameter to the other method.

Reply:
but it breaks backward compatibility

Reply:
It shouldn't. Default parameters are handled by the caller, not the callee. Have you tried?

Reply:
Yes. That's why I have the issue.  There is an unit test for that. You can try to remove the empty parameter overload and compile